.nvmrc

```diff
@@ -1 +1 @@
-12.22.1
+12.14.0
```

Comment:
Is the change to v12.14.0 intentional? I have a vague recollection we've been upgrading to v12.22.1 as of late.